src/ohri-form.component.test.tsx

```diff
@@ -326,6 +329,62 @@ describe("OHRI Forms:", () => {
       await act(async () => expect(bsaField.value).toBe("2.24"));
     });
 
+    it("Should evaluate Weight for Height Zscore result", async () => {
+      // setup
+      await act(async () => renderForm(null, wh_zscore));
+
+      const heightField = await findNumberInput(screen, "Height");
+      const weightField = await findNumberInput(screen, "Weight");
+      await act(async () => expect(heightField.value).toBe(""));
+      await act(async () => expect(weightField.value).toBe(""));
+
+      // replay
+      fireEvent.blur(heightField, { target: { value: 110 } });
+      fireEvent.blur(weightField, { target: { value: 45 } });
+
+      // verify
+      await act(async () => expect(heightField.value).toBe("110"));
```

Comment:
Here it looks like all you are asserting is that the height and weight fields have value. Don't you need to assert that the `WeightForHeightZscore` field has the right value given the passed height and weight values?

Reply:
@kajambiya in this case the average height is 110, we also don't know what the user inputs because it is no explicit